app/models/member.rb

```diff
@@ -6,9 +6,7 @@ class Member < ActiveRecord::Base
   before_save { self.email.try(:downcase!) }
 
   def self.find_from_request(akid: nil, id: nil)
-    if akid.present?
-      actionkit_user_id = AkidParser.parse(akid)[:actionkit_user_id]
-      #member = find_by(actionkit_user_id: actionkit_user_id)
+    if actionkit_user_id = AkidParser.parse(akid, Settings.action_kit.akid_secret)[:actionkit_user_id]
```

Comment:
I would really prefer not putting assignment in conditionals - it's hard to tell if its a bug